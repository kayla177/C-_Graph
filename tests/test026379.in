ENTITY GLMHTMNB name_GLMH journal
ENTITY V5W7HQY1 name_V5W7 paper
ENTITY QNVET9Z0 name_QNVE journal
ENTITY YJ2Q6FXH name_YJ2Q institution
ENTITY X3DA3OGE name_X3DA paper
ENTITY BFWGFLD2 name_BFWG author
ENTITY TTPLS9YD name_TTPL conference
ENTITY FL1UZ3O9 name_FL1U journal
ENTITY DUR0C31Z name_DUR0 institution
ENTITY CB5TTO7F name_CB5T author
RELATIONSHIP YJ2Q6FXH collaborated_with TTPLS9YD 9.6
RELATIONSHIP GLMHTMNB authored_by X3DA3OGE 6.9
RELATIONSHIP DUR0C31Z presented_at CB5TTO7F 0.4
RELATIONSHIP BFWGFLD2 authored_by X3DA3OGE 9.6
RELATIONSHIP BFWGFLD2 cited_by TTPLS9YD 8.7
RELATIONSHIP X3DA3OGE presented_at YJ2Q6FXH 5.5
RELATIONSHIP BFWGFLD2 authored_by YJ2Q6FXH 8.1
RELATIONSHIP DUR0C31Z affiliated_with GLMHTMNB 2.0
RELATIONSHIP BFWGFLD2 cited_by QNVET9Z0 3.5
RELATIONSHIP GLMHTMNB affiliated_with BFWGFLD2 6.3
RELATIONSHIP FL1UZ3O9 presented_at X3DA3OGE 7.1
RELATIONSHIP YJ2Q6FXH authored_by QNVET9Z0 9.8
RELATIONSHIP DUR0C31Z collaborated_with FL1UZ3O9 3.5
RELATIONSHIP CB5TTO7F affiliated_with YJ2Q6FXH 9.1
RELATIONSHIP FL1UZ3O9 presented_at YJ2Q6FXH 7.8
DELETE TTPLS9YD
HIGHEST
PRINT V5W7HQY1
PATH X3DA3OGE QNVET9Z0
DELETE V5W7HQY1
PRINT BFWGFLD2
DELETE X3DA3OGE
HIGHEST
PRINT CB5TTO7F
PRINT GLMHTMNB
EXIT
