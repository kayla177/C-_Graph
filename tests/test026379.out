success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
GLMHTMNB QNVET9Z0 24.2

X3DA3OGE YJ2Q6FXH QNVET9Z0 15.3
success
X3DA3OGE YJ2Q6FXH QNVET9Z0 GLMHTMNB
success
GLMHTMNB QNVET9Z0 24.2
DUR0C31Z YJ2Q6FXH
DUR0C31Z BFWGFLD2
